docs/pages/installation.md

```diff
@@ -185,6 +185,17 @@ After you selected "Foundation for Sites", Foundation CLI will ask you which tem
   <p>To remove the Foundation 5 CLI, run <code>gem uninstall foundation</code>. After testing the Foundation 6 CLI, if you want to remove it to go back to the old CLI, run <code>npm uninstall foundation-cli --global</code>.</p>
 </div>
 
+<div class="callout info">
+  <p><strong>Windows users</strong>: make sure you've python v2.7 available in your node environment since it's required by the node-gyp tool. There are two way to achieve this</p>
+
+  <ol>
+    <li>Install the <a href="https://github.com/felixrieseberg/windows-build-tools">windows-build-tools</a> (recommended) and make python afterwards accessible via <code>npm config set python "%USERPROFILE%\.windows-build-tools\python27\python.exe"</code></li>
```

Comment:
Never had to do this step with npm (afaik).

Reply:
> I had probably used `--add-python-to-path` – @DanielRuf 

Yup I think the same.
But afaik it adds the installed python to your system environment variables and therefore affects more than just your node environment.

Imo it feels safer to limit it to node as the user (probably) only installs it for this environment.

Reply:
> But afaik it adds the installed python to your system environment variables and therefore affects more than just your node environment.

Which is ok as most solutions only work with Python 2. I have it on my path variable and never had any issues.

Reply:
> Imo it feels safer to limit it to node as the user (probably) only installs it for this environment.

Well, actually it installs much more which will be on the path then, eg the Windows build tools / compiler toolchain and so n.

https://github.com/felixrieseberg/windows-build-tools/blob/master/ps1/launch-installer.ps1#L37

Reply:
Also what about yarn, pnpm and other solutions which might need it then?

Reply:
> Which is ok as most solutions only work with Python 2. I have it on my path variable and never had any issues. – @DanielRuf 

With 'most solutions' you mean other node packages?
I'm still not convinced it's the right way to add python 2.7 to the system environment variables in the course of the windows-build-tools installation.

Therefore I've tested again a scenario on a new machine with only node.js and (this time) python 3.7 installed.
I did install the windows-build-tools with `--add-python-to-path` but afterwards `python --version` was still showing 3.7. So `foundation new` did fail because it requires python 2.x – not python 3.x

Imagine users, that are not very familiar with node and have already python 3.x installed (doesn't necessarily mean they are python devs!), who follow an instruction with `--add-python-to-path`.
Pretty sure they end up frustrated and spend some time to find out what is wrong.

The solution in this PR (via `npm set python`) goes around this problem and doesn't mess up with an existing python installation.
So I think this is what we should recommend. What do you think?

> Also what about yarn, pnpm and other solutions which might need it then? – @DanielRuf 

You mean providing more help for these tools, too?
I don't think it's necessary since npm is the recommended tool and the others are just an option for people who are already familiar with it. So a one line installation should be fine here.

Reply:
> With 'most solutions' you mean other node packages?

(Python) projects in general which require Python.

Reply:
> via npm set python

Please do not forget Yarn and others ;-)

Reply:
> I don't think it's necessary since npm is the recommended tool and the others are just an option for people who are already familiar with it. So a one line installation should be fine here.

I don't agree here.

Also both can be on the PATH. The npm way is just some "hack" in my opinion.

Reply:
> I don't think it's necessary since npm is the recommended tool and the others are just an option for people who are already familiar with it. So a one line installation should be fine here.

I don't agree here.

The npm way is just some "hack" in my opinion.

I will test it this week. Also macOS has Python 2.7 by default.

Reply:
Well, I guess we can use the text that you wrote but I guess there will be some cases where there will be problems.

In my opinion we should refer to the docs of node-gyp where all these solutions are mentioned and linked like windows-build-tools.

My longterm goal at work is to remove node-gyp and Python as required dependencies.

Reply:
Then what do you suggest?
`npm install --global --production --add-python-to-path windows-build-tools` does **not** work (as mentioned above)

> (Python) projects in general which require Python.

It makes zero sense to me to install python 2.x via the windows-build-tools if the desired python projects require 3.x

> Please do not forget Yarn and others ;-)

For yarn it's almost the same (`yarn config set`).
However I've realized the cli does always use npm so we may ignore **all** other tools.

Reply:
> if the desired python projects require 3.x

I know none.

> However I've realized the cli does always use npm so we may ignore all other tools.

It's not about the CLI, also the normal process / dev for contributors requires the same dependencies ;-)

We can merge it as is for now.
I'm just too petty, you can ignore what I wrote.

Reply:
This issue is only about the CLI (I've adjusted only the section about the CLI)

> We can merge it as is for now.
> I'm just too petty, you can ignore what I wrote.

No problem, I know this ambition (to achieve a perfect result) quite well 😉  
When you've got an idea about improving the docs regarding the other tools feel free to create a PR and add me as reviewer.